CHANGELOG.md

```diff
@@ -7,7 +7,8 @@ in 0.1 minor versions.
 
 To get the diff for a specific change, go to https://github.com/superdesk/web-publisher/commit/XXX where XXX is the change hash
 
-* 0.5.0
+* 0.5.0-3
```

Comment:
we should start new part for 0.6.0 with new features (as it's targeted next release).